src/testing/time_refinement_study.cpp

```diff
@@ -23,11 +23,17 @@ TimeRefinementStudy<dim, nstate>::TimeRefinementStudy(
 template <int dim, int nstate>
 Parameters::AllParameters TimeRefinementStudy<dim,nstate>::reinit_params_and_refine_timestep(int refinement) const
 {
-     PHiLiP::Parameters::AllParameters parameters = *(this->all_parameters);
-     
-     parameters.ode_solver_param.initial_time_step *= pow(refine_ratio,refinement);
-     
-     return parameters;
+    PHiLiP::Parameters::AllParameters parameters = *(this->all_parameters);
+
+    parameters.ode_solver_param.initial_time_step *= pow(refine_ratio,refinement);
+
+    //For RRK, do not end at exact time because of how relaxation parameter convergence is calculatd
```

Comment:
Typo at the end of comment.

Reply:
Thanks for pointing that out. Changing in the next commit.